src/s1reader/utils/plot_bursts.py

```diff
@@ -0,0 +1,172 @@
+import os
+import argparse
+import glob
+from importlib import import_module
```

Comment:
```suggestion
import argparse
import glob
from importlib import import_module
import os
```
Nitpick: alphabetical order for readability